eng/pipelines/common/global-build-job.yml

```diff
@@ -31,6 +31,9 @@ jobs:
     timeoutInMinutes: ${{ parameters.timeoutInMinutes }}
     workspace:
       clean: all
+    ${{ if ne(parameters.enableRichCodeNavigation, '') }}:
+      enableRichCodeNavigation: true
+      richCodeNavigationLanguage: csharp,cpp
```

Comment:
should we allow this to be passed as a parameter when referencing global-build-job? For example the `libs.refs+libs.src` job is not enabling it for cpp yet.

Reply:
Oh great point, good catch! I'll update, thanks